src/main/java/uk/ac/ebi/ena/readtools/loader/fastq/IlluminaSpot.java

```diff
@@ -11,9 +11,10 @@
 package uk.ac.ebi.ena.readtools.loader.fastq;
 
 
+import uk.ac.ebi.ena.readtools.loader.common.consumer.DataConsumable;
 
-public class 
-IlluminaSpot
+public class
+IlluminaSpot implements DataConsumable

```

Comment:
Rename IluminaSpot to FastqSpot please. Remove DataConsumable interface and replace with FastSpot.